catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.less

```diff
@@ -16,6 +16,13 @@
         width: auto;
         overflow: hidden;
         text-overflow: ellipsis;
+        margin-right: 9rem;
```

Comment:
We should use the vars defined in `vars.less`.

That includes `@minimumSpacing`, `@mediumSpacing`, and `@largeSpacing`.